css/tablesort.css

```diff
@@ -0,0 +1,31 @@
+th[role=columnheader]:not(.no-sort) {
+    cursor: pointer;
+}
+
+th[role=columnheader]:not(.no-sort):after {
+    content: '';
```

Comment:
```suggestion
    content: '▼';
```

Did you try using unicode arrows for that instead of the border-arrow?
_(Just a suggestion, feel free to close)_

Reply:
Was using the standard styles from tristen-tablesort but this may make it align vertically better. I'll give it a try

Reply:
The latest commit https://github.com/contao-components/tristen-tablesort/pull/1/commits/cc89204b67d52135f290ef82546440c49a000456 uses these

Ascending: ▴ instead of ▲
Descending: ▾ instead of ▼

![image](https://github.com/contao-components/tristen-tablesort/assets/55794780/21a0b8b3-8d93-4bd5-9623-f52d96250899)

Your suggestion is kinda too big I think:
![image](https://github.com/contao-components/tristen-tablesort/assets/55794780/6ceb6185-4f40-4935-83fb-968fc35db421)

Reply:
Wdyt @ausi ?

Reply:
Perfect 👌